oic.devicemap-content.json

```diff
@@ -237,6 +237,14 @@
       {"resourcetypetitle":"Energy Generation", "resourcetypeid": "oic.r.energy.generation"}
     ]
   },
+  {
+    "devicename": "Energy Monitor",
+    "devicetype": "oic.d.energymonitor",
+    "vertical":   "Smart Home",
+    "resources": [
+      {"resourcetypetitle":"Any consumption resource type", "resourcetypeid": "oic.r.*.consumption"}
```

Comment:
can we handle wild cards in CTT?

Reply:
We already use <x> as a wildcard elsewhere (for generic sensor); so changed the '*' to that.